demos/squares/index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <title>Squares</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+        <style>
+        * {
+          margin: 0;
+        }
+        </style>
+    </head>
+    <body>
+        <div id="app"></div>
+        <script src="./index.tsx"></script>
```

Comment:
Not sure why you're using `.tsx` here.  You aren't using TypeScript or JSX, and even if you were, they'd need to be converted to JavaScript before being included in a page.

Reply:
The dev system transpiles it to a js when running `yarn dev`. I just copied from the other demo.